UnityProject/Assets/Scripts/Core/UniversalObjectPhysics.cs

```diff
@@ -2141,12 +2141,11 @@ public static float SizeToWeight(Size size)
 
 	// netid of the game object we are buckled to, NetId.Empty if not buckled
 	[SyncVar(hook = nameof(SyncBuckledToObject))]
-	protected UniversalObjectPhysics ObjectIsBuckling = null;
+	public UniversalObjectPhysics ObjectIsBuckling = null;
```

Comment:
we shouldn't let anyone change the value of this except inside the sync method, can you make it a property with private setter?

like this:
```suggestion
    [field: SyncVar(hook = nameof(SyncBuckledToObject))]
	public UniversalObjectPhysics? ObjectIsBuckling { get; private set; }
```

Reply:
done